client/shipping-label/views/purchase/steps/packages/add-item.js

```diff
@@ -72,7 +72,7 @@ const AddItemDialog = ( {
 				<h1 className="form-section-heading">{ __( 'Add item' ) }</h1>
 				<div className="wcc-label-packages-dialog__body">
 					<p>
-						{ __( 'Which item would you like to add to {{pckg/}}?', {
+						{ __( 'Which items would you like to add to {{pckg/}}?', {
```

Comment:
This should be a single/plural case based on the number of items available to move.